content/docs/15.how-to-guides/azure-workload-id.md

```diff
@@ -0,0 +1,209 @@
+
+## Variables
+
+Define the following variables and update them to match your environment.
+
+```shell
+# Managed User Identity Name
+ID_NAME="kestra-managed-user"
+# Azure Resource Group
+RESOURCE_GROUP="demo"
+# Physical location you wish to provision resources
+LOCATION="eastus"
+# The name of your Azure Kubernetes Cluster
+AKS_NAME="demo-cluster"
+# The name of your Azure Key Vault
+KEYVAULT_NAME="my-demo-vault"
+# The name you wish to provide to the Kubernetes Service Account linked to the managed identity
+SERVICE_ACCOUNT_NAME="kestra-sa"
+# The namespace to deploy the service account. Use the same location as your Kestra deployment
+SERVICE_ACCOUNT_NAMESPACE="default"
+# The Federated ID credential for linking the OIDC issuer to the service account
+FEDERATED_IDENTITY_CREDENTIAL_NAME="kestra-fed-cred"
+```
+
+## Create the resources
+
+To start, create the 3 main resources, the Key Vault, the managed identity and the AKS cluster. Once these have been provisioned, there are several identifiers we must capture for later use.
+
+### Azure Key Vault
+
+```shell
+az keyvault create \
+  --name $KEYVAULT_NAME \
+  --resource-group $RESOURCE_GROUP \
+  --location $LOCATION 
+```
+
+### Managed Identity
+
+```shell
+az identity create --name $ID_NAME \
+ --resource-group $RESOURCE_GROUP
+```
+
+### AKS Cluster
+
+```shell
+az aks create \
+  --resource-group $RESOURCE_GROUP \
+  --name $AKS_NAME \
+  --enable-oidc-issuer \
+  --enable-workload-identity \
+  --node-count 1 \
+  --generate-ssh-keys
+```
+
+### Setting identifiers from new resources
+
+Once all the above have been created, capture the following information in variables for use later on
+
+```shell
+OBJECT_ID=$(az identity show --name $ID_NAME --resource-group $RESOURCE_GROUP --query 'principalId' --output tsv)
+MANAGED_CLIENT_ID=$(az identity show --name $ID_NAME --resource-group $RESOURCE_GROUP --query clientId --output tsv)
+AKS_OIDC_ISSUER="$(az aks show --name "${AKS_NAME}" --resource-group "${RESOURCE_GROUP}" --query "oidcIssuerProfile.issuerUrl" --output tsv)"
+```
+
+## Link Identity Resources
+
+One of the more challenging aspects of this setup is correctly linking together the various resources. This section covers how to tie the managed identities to the resources to allow access by the Kestra application. 
+
+### Create role assignment for created user 
+
+This is one of the more critical steps as it sets the permission the resource has on the Key Vault. As Kestra needs to read and write secrets to the vault, the "Key Vault Secrets Officer" provides least priviledged access for this operation. Further details on this role can be found [here](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli#azure-built-in-roles-for-key-vault-data-plane-operations). 
```

Comment:
```suggestion
This is one of the most critical steps as it sets the permission the resource has on the Key Vault. As Kestra needs to read and write secrets to the vault, the "Key Vault Secrets Officer" provides least priviledged access for this operation. Further details on this role can be found [in Azure's RBAC guide](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli#azure-built-in-roles-for-key-vault-data-plane-operations). 
```
useful to label links other than "here" so that in case the link is broken, people know what was meant